ts/routes/deck-options/deck-options-base.scss

```
@@ -28,6 +30,8 @@ input {
 // Setting 100% height causes the sticky element to hide as you scroll down on Safari.
 html {
     height: initial;
+    -webkit-user-select: none;
+    user-select: none;
 }
```

```suggestion
}

@media (pointer: coarse) {
  html {
      -webkit-user-select: none;
      user-select: none;
  }
}
```

Maybe something like this? (untested)

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer

---

that would also select people using keyboard/voice assist, so people will see inconsistent behaviour in a laptop vs a PC.

(actually i still think disabling this everywhere is a better choice, mainly for consistency's sake)

---

LGTM